code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -225,15 +225,15 @@
 
 /obj/machinery/portable_atmospherics/canister/halon
 	name = "Halon canister"
-	desc = "Halon, remove oxygen from high temperature fires and cool down the area"
+	desc = "Halon, removes oxygen from high temperature fires and cools down the area"
 	icon_state = "halon"
 	base_icon_state = "halon"
 	gas_type = /datum/gas/halon
 	filled = 1
 
 /obj/machinery/portable_atmospherics/canister/hexane
 	name = "Hexane canister"
-	desc = "hexane, highly flammable"
+	desc = "Hexane, useful for removing contaminants."
```

Comment:
This one was a big change from old to new. Can you explain this one for me? Is it useful for removing contaminants because IT PURGES THEM WITH FIRE or was the old comment merely wrong/misleading/out of date?

Reply:
See previous review from ghilker. The old description was a copy paste error from the hydrogen canister description.